src/integration/pages/storyPage/crossPlatformTests.js

```diff
@@ -3,17 +3,28 @@ import { runCommonCrossPlatformTests } from '../../common';
 export default () => {
   runCommonCrossPlatformTests();
 
-  it('I can see an image with a caption', () => {
-    const imageEl = document.querySelector(
-      'main figure img, main figure amp-img',
-    );
-    expect(imageEl).toBeInTheDocument();
+  const noscripts = document.querySelectorAll('noscript');
+  const firstImageLazyloaded = noscripts.length > 1;
 
-    const imageCaptionEl = document.querySelector('main figure figcaption');
-    expect(imageCaptionEl).toBeInTheDocument();
-    expect(imageCaptionEl.textContent).toBeTruthy();
-    expect(imageCaptionEl.textContent).toMatchSnapshot();
-  });
+  if (firstImageLazyloaded) {
+    it('Lazy loaded image contained in noscript tag', () => {
+      const firstImage = noscripts[1];
+      expect(firstImage.innerHTML).toMatchSnapshot();
```

Comment:
```suggestion
      expect(firstImage).toMatchSnapshot();
```
The snapshot doesn't currently include the `nonscript` tag, it might be good to include that in the snapshot too, I _think_ using `innerHTML` removes it?